### PR TITLE
OCPQE-18321: Invoke e2e test on HyperShift-MCE-Agent-dualstack

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-baremetalds-assisted-hypershift-agent-mce-dualstack-mgmt-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-baremetalds-assisted-hypershift-agent-mce-dualstack-mgmt-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-baremetalds-assisted-hypershift-agent-mce-dualstack-mgmt-f7

**Comment by LiangquanLi930** [comment]
@liangxia Could you help to review this PR? Thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -2407,7 +2407,33 @@ tests:
...
+        IP_STACK=v4v6
+        NETWORK_TYPE=OVNKubernetes
+        VM_EXTRADISKS=true
+        VM_EXTRADISKS_LIST=vda
+        VM_EXTRADISKS_SIZE=1024G
+        NUM_EXTRA_WORKERS=3
+        EXTRA_WORKER_VCPU=8
+        EXTRA_WORKER_MEMORY=16384
+        EXTRA_WORKER_DISK=100
+      IP_STACK: v4v6
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:2426]: Why do we need the same info `IP_STACK` twice ?

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:2426]: They are used for different steps, https://github.com/openshift/release/blob/45a1fafbbd842daff9b791650bf91d2d887125bd/ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14__periodics.yaml#L141

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:2426]: It indicates something we can improve in the design in the future.

Not a blocker to this PR though.

**Pull request merged**